js/src/pages/LoginPage.jsx

```diff
@@ -1,11 +1,14 @@
-// src/components/auth/LoginForm.jsx
 import React, { useState } from 'react';
-import { signIn } from 'aws-amplify/auth';
-import { Box, TextField, Button, Typography, Alert, LoadingSpinner } from '../components/index';
+import { signIn, fetchUserAttributes } from 'aws-amplify/auth';
+import { useNavigate } from 'react-router-dom';
+import { Box, TextField, Button, Typography, Alert, CircularProgress } from '@mui/material';
```

Comment:
why change back to mui?
Get them from components folder

Reply:
changed back to components